src/Network/Send/kRO/Sakexe_0.pm

```diff
@@ -1367,434 +1381,4 @@ sub sendFriendRemove {
 # 0x0208,11,friendslistreply,2:6:10
 # sendFriendReject:0/sendFriendAccept:1
 
-1;
```

Comment:
when i fast searching a packet meaning using notepad++ , many times the results is these lines you want to delete
So i vote to not to delete , as a document

Reply:
what about update openkore\tables\packetdescriptions.txt?

in my point of view this piece of code is unnecessary in this file, some of this packets dont have description or are receive packets

Reply:
if it's already in packetdescriptions.txt then it's fine
i remember asked these lines on forum , why we have these line almost in every ST and a moderator in forum said as a document (or something similar)

Reply:
the better is keep all in one file